.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,42 +0,0 @@
-_PR description_
-
-### Contents
-
-- _item_
-
-### Rationale
-
-_design decisions and extended information_
-
-
-<hr>
-
-## How to fill a PR description (remove this)
-
-Please give a concise description of your PR.
-
-The target readers could be future developers, reviewers, and auditors. By reading your description, they should easily understand the changes proposed in this pull request.
-
-MUST: Reference the issue to resolve
```

Comment:
Can we add back the guidance on how to write the description?

Reply:
Definitely. Thanks for the correction.